.github/workflows/renovate.yaml

```diff
@@ -30,6 +30,30 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 5
     steps:
+      - name: "Calculate effective token"
+        env:
+          EFFECTIVE_TOKEN: "${{ secrets.KROXYLICIOUS_GITHUB_READWRITE_TOKEN || secrets.GITHUB_TOKEN }}"
+          KROXYLICIOUS_GITHUB_READWRITE_TOKEN_SET: "${{secrets.KROXYLICIOUS_GITHUB_READWRITE_TOKEN  && 'true' || 'false' }}"
+        run: |
+          echo "EFFECTIVE_TOKEN=${EFFECTIVE_TOKEN}" >> $GITHUB_ENV
+          echo "KROXYLICIOUS_GITHUB_READWRITE_TOKEN_SET=${KROXYLICIOUS_GITHUB_READWRITE_TOKEN_SET}" >> $GITHUB_ENV
+      - name: "Set actor from secret"
+        if: env.KROXYLICIOUS_GITHUB_READWRITE_TOKEN_SET == 'true'
```

Comment:
The intent is that the workflow works for developers on their forks without the KROXYLICIOUS_GITHUB_READWRITE_TOKEN secret being set. This allows the developer to develop the workflow with minimal bother.

Unfortunately you can't use the /user to interrogate the `secrets.GITHUB_TOKEN`, so it needs to branch.